eth/tracers/js/tracer_test.go

```diff
@@ -227,37 +227,6 @@ func TestIsPrecompile(t *testing.T) {
 		t.Errorf("Tracer should consider blake2f as precompile in istanbul")
 	}
 
-	// test sha disabled in archimedes
```

Comment:
Why do we need to remove these?

Reply:
there is info about it in [notion page](https://www.notion.so/scrollzkp/l2geth-failing-tests-f5f8b38b2d4047a3b4f55e01770d3455?pvs=4#066c13f87fa4488a8e59882e75f0e2f4)